packages/network-controller/src/NetworkController.ts

```diff
@@ -788,9 +786,9 @@ function validateNetworkControllerState(state: NetworkState) {
 
   if (!networkClientIds.includes(state.selectedNetworkClientId)) {
     throw new Error(
-      `NetworkController state is invalid: \`selectedNetworkClientId\` ${inspect(
-        state.selectedNetworkClientId,
-      )} does not refer to an RPC endpoint within a network configuration`,
+      // False negative - this is a string.
```

Comment:
Nit: Shouldn't all of these be called "false positives" rather than "false negatives"? 😅

The lint rule is flagging something that it shouldn't, so this is a "positive" result.

Reply:
Yeah, I suppose so. Fixed in [557c1a4](https://github.com/MetaMask/core/pull/3672/commits/557c1a4a31e9e33e9fe8f2a7c063901fb1798856).